web/beacon-app/src/features/members/components/ChangePassword/ChangePasswordForm.tsx

```diff
@@ -0,0 +1,50 @@
+import { t, Trans } from '@lingui/macro';
+import { Button } from '@rotational/beacon-core';
+import { ErrorMessage, Form, FormikHelpers, FormikProvider } from 'formik';
+
+import PasswordField from '@/components/ui/PasswordField';
+
+import { useChangePasswordForm } from '../../types/changePasswordFormService';
+
+type ChangePasswordForm = {
+  handleSubmit: (values: any, helpers: FormikHelpers<any>) => void;
+  initialValues: any;
+};
+
+function ChangePasswordForm({ handleSubmit, initialValues }: ChangePasswordForm) {
+  const formik = useChangePasswordForm(handleSubmit, initialValues);
+
+  const { getFieldProps, isSubmitting } = formik;
+
+  return (
+    <FormikProvider value={formik}>
+      <Form className="space-y-3">
+        <PasswordField
+          labelClassName="font-bold"
+          label={t`Enter new password`}
+          fullWidth
+          {...getFieldProps('new_password')}
+        />
+        <ErrorMessage name="new_password" component="small" className="text-xs text-danger-500" />
```

Comment:
Will using `small` make the text size applied by Tailwind smaller? Any text size smaller than `xs` could be difficult for users to read.